math/generic/include/algebra/math/generic.hpp

```diff
@@ -8,6 +8,8 @@
 #pragma once
 
 // Impl include(s).
+#include "algebra/math/boolean.hpp"
+#include "algebra/math/common.hpp"
```

Comment:
Same question here. Why do we need `boolean.hpp` here?

I assume `common.hpp` was missing earlier, right? Since none of the current changes justify its addition either.